data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -65,7 +66,7 @@ class TokenRepository(
             TokenType.ETHER,
             ZERO_ADDRESS,
             18,
-            "ETH",
+            BuildConfig.NATIVE_CURRENCY_SYMBOL,
             "Ether",
             "local::ethereum"
```

Comment:
Maybe changing this string to `local::native_token` and adding a different resource for the xdai build would make sense